src/cov.jl

```diff
@@ -29,6 +29,21 @@ scattermat_zm(x::DenseMatrix, wv::WeightVec, vardim::Int) =
     _symmetrize!(Base.unscaled_covzm(x, _scalevars(x, values(wv), vardim), vardim))
 
 if VERSION < v"0.5.0-dev+679"
+"""
+    scattermat(x, [wv,] [mean=nothing,] [vardim=1]) -> Matrix
+
+Compute the scatter matrix, optionally specifying a weighting vector.
+The function accepts two optional keyword arguments:
+
+  • `mean`: a known mean value. By default it is set to `nothing`,
```

Comment:
Instead of saying "it is set to `nothing", just write the `=nothing` after the argument name.